pytest_astropy_header/display.py

```diff
@@ -71,14 +68,13 @@ def pytest_report_header(config):
     args = config.args
 
     # TESTED_VERSIONS can contain the affiliated package version, too
-    if len(TESTED_VERSIONS) > 1:
-        for pkg, version in TESTED_VERSIONS.items():
-            if pkg not in ['Astropy']:
-                s = "\nRunning tests with {} version {}.\n".format(
-                    pkg, version)
-    else:
-        s = "\nRunning tests with Astropy version {}.\n".format(
-            TESTED_VERSIONS['Astropy'])
+    skip_astropy = len(TESTED_VERSIONS) > 1 and 'Astropy' in TESTED_VERSIONS
+
+    s = ''
+    for pkg, version in TESTED_VERSIONS.items():
```

Comment:
Now that I look at this, I actually don't understand why special treatment for `'Astropy'` or why we allow multiple entries in `TESTED_VERSIONS`.

* Why can't astropy use this plugin the same way as other packages, by specifying its own `TESTED_VERSIONS` in its `conftest.py`?
* Is there a case where someone would test more than one package all at once in pytest?

Reply:
It's because `'Astropy'` is defined by default in `TESTED_VERSIONS`. So if another package wants to print its package name instead (and some added also astropy-helpers), they add their name to `TESTED_VERSIONS`, which is detected here by the `len(TESTED_VERSIONS) > 1` ...
A better option would be not to add Astropy by default.